backend/services/interfaces/ICourseService.ts

```diff
@@ -5,6 +5,8 @@ export interface CourseRequestDTO {
   image: string;
```

Comment:
Do we want people creating the course to have control over the private/public states when created/updated? I don't 100% remember the flow, but I'm thinking that when a course is created it defaults to not published.

cc: @draymond63